packages/react-scripts/scripts/utils/createJestConfig.js

```diff
@@ -45,6 +45,9 @@ module.exports = (resolve, rootDir) => {
     moduleNameMapper: {
       '^react-native$': 'react-native-web',
     },
+    globals: {
+      __TS_CONFIG__: resolve('template/tsconfig.json'),
```

Comment:
I've tried that already, but again it's problematic in terms of the project root in use during the tests.
When running the first test (L127 of e2e-simple.sh), `tsconfigPath` determined the way you mentioned above would reference
```
create-react-app-typescript/packages/react-scripts/tsconfig.json
```
since the `react-scripts` folder is the current directory when running this test. Apparently, this file does not exist - the `tsconfig.json` is only present in the template.

Again, it might be required to use different paths here. Trying to figure out how.